bot/exts/fun/user_events.py

```diff
@@ -0,0 +1,603 @@
+        return status
+
+    @property
+    def developers_role(self) -> Role:
+        """Return guild developers role."""
+        return self.guild.get_role(Roles.verified)
+
+    @property
+    def user_event_ongoing_role(self) -> Role:
+        """Return guild user-event-ongoing role."""
+        return self.guild.get_role(Roles.user_event_ongoing)
+
+    @property
+    def guild(self) -> Guild:
+        """Return guild instance."""
+        return self.bot.get_guild(Server.id)
+
+    @property
+    def user_event_coord_channel(self) -> TextChannel:
+        """Return #user-events-coordinators channel."""
+        return self.bot.get_channel(Channels.user_event_coordinators)
+
+    @property
+    def user_event_announcement_channel(self) -> TextChannel:
+        """Return #user-events-announcement channel."""
+        return self.bot.get_channel(Channels.user_event_announcements)
+
+    @property
+    def user_events_list_channel(self) -> TextChannel:
+        """Return #user-events-list channel."""
+        return self.bot.get_channel(Channels.user_event_list)
+
+    @property
+    def user_event_voice_channel(self) -> VoiceChannel:
+        """Return #user-events-voice channel."""
+        return self.bot.get_channel(Channels.user_event_voice)
+
+    @staticmethod
+    def user_event_embed(event_name: str, event_description: str, organizer: Member, status: str) -> Embed:
+        """Embed representing a user event."""
+        embed = Embed(
+            title=f"{event_name}",
+            description=(
+                f"Organizer: {organizer.mention}\n\n"
+                f"{event_description}\n\n"
+                f"**Status:** {status}"
+            )
+        )
+        return embed
+
+    async def fetch_subscribers(self, message_id: int) -> list:
+        """Fetch reacted users to event message as subscribers."""
+        # Fetch the event message
+        message = await self.user_events_list_channel.fetch_message(message_id)
+
+        reaction = message.reactions[0]
+
+        # Flatten into a list
+        users = await reaction.users().flatten()
+        return users
+
+    async def sync_subscribers(self, event_name: str, sub_ids: list) -> None:
+        """Sync subscribers with the site."""
+        data = {
+            "subscriptions": sub_ids
+        }
+        await self.bot.api_client.patch(
+            f"bot/user-events/{event_name}",
+            data=data
+        )
+
+    async def update_user_event_message(self, status: str, user_event: dict) -> None:
+        """Update event message on #user-events-list channel."""
+        # Fetch user event message
+        message = await self.user_events_list_channel.fetch_message(user_event["message_id"])
+
+        embed = self.user_event_embed(
+            user_event["name"],
+            user_event["description"],
+            self.guild.get_member(user_event["organizer"]),
+            status
+        )
+        embed.set_footer(text="React to be notified.")
```

Comment:
I think `React to get notified about event start.` would be better and give more information.